shopno/src/styles/style.scss

```
@@ -0,0 +1,594 @@
+@import "./fonts.scss";
+@import "./popUp.scss";
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 10px;
+    @media (max-width: 1024px) {
```

It's not recommended to mix common styles with media queries. Put out of blocks your `@media`.

---

Remove all nested media query
```suggestion
    @media (max-width: 51.2em) {
    html {
        font-size: 55%;
    }
}
@media (max-width: 87.5em) {
    html {
        font-size: 50%;
    }   
}
```